lib/src/room.dart

```diff
@@ -2130,7 +2130,7 @@ class Room {
       : setSpaceChild(roomId, via: const []);
 
   @override
-  bool operator ==(dynamic other) => (other is Room && other.id == id);
```

Comment:
Can you explain a little bit about this case?

Reply:
`dynamic` can be null but the operator `==` is not nullable (`bool operator ==`), so the new version of dart require the usage of `Object` here instead to be null safe

Reply:
if you could provide a link, it would be great

Reply:
https://dart.dev/tools/linter-rules/null_check_on_nullable_type_parameter

Reply:
Also the related error from CI: https://github.com/linagora/twake-on-matrix/actions/runs/9273371007/job/25513989168